src/lib.rs

```diff
@@ -1,16 +1,22 @@
 #![feature(test)]
 
+#[macro_use]
```

Comment:
Guess we should be good to remove those `extern-crates` as it's covered by the compiler.